provider/marathon_test.go

```diff
@@ -551,7 +551,7 @@ func TestMarathonTaskFilter(t *testing.T) {
 					},
 				},
 			},
-			expected:         false,
+			expected:         true,
```

Comment:
I made this change. I wasn't sure how these tests work but this passes now. Does it read the application section then expect the task section?